src/application/extension-popup-menu/popup-home-view/app.tsx

```diff
@@ -0,0 +1,75 @@
+import { useNavigate } from 'react-router-dom';
+
+import {
+  AGORA_LOGO,
+  POLYMARKET_LOGO,
+  SNAPSHOT_LOGO,
+  TALLY_LOGO,
+} from 'shared/web3';
+import { IDRISS_ICON } from 'shared/idriss';
+
+import { TRADING_COPILOT_LOGO } from '../constants';
+import { EXTENSION_POPUP_MENU_ROUTES } from '../app';
+
+import { SectionTile } from './components';
+
+export const App = () => {
+  const navigate = useNavigate();
+  return (
+    <div className="flex max-w-[570px] grow flex-col items-center bg-gray-100 px-1 pt-6">
+      <header className="mb-6 text-center">
+        <h1 className="mb-2 text-2xl font-bold text-black">
+          Hello, it&apos;s your web3 copilot speaking
```

Comment:
cool trick to avoid `&apos;` and other things like that is to do:
```tsx
{`Hello, it's your web3 copilot speaking`}
```

Reply:
thanks, replaced 👌